<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmLogOutWidget</class>
 <widget class="QFrame" name="CArmLogOutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>193</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>193</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>193</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #090E10;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>59</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>59</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>29</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>49</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>49</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-image: url(:/CArmWorkStation/ImageSource/LogOut/user.png);
background-position:center left;
background-repeat: no-repeat;
background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userLabel">
        <property name="minimumSize">
         <size>
          <width>148</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>148</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-color: transparent;
font: 16px &quot;Microsoft YaHei&quot;;
color:#B4C4D9;
border:0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>59</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>59</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color: #B4C4D9;
padding:0px;
margin:-1px;
font: 16px &quot;Microsoft YaHei UI&quot;;
}
QPushButton:hover
{
color: rgb(255, 255, 255);
}
</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">width:297px;
height:1px;
background:rgba(255,255,255,0.1);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>29</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>49</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>49</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel
{
background-image: url(:/CArmWorkStation/ImageSource/LogOut/set.png);
background-position:center left;
background-repeat: no-repeat;
background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>65</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color: #B4C4D9;
padding:0px;
margin:-1px;
font: 16px &quot;Microsoft YaHei UI&quot;;
text-align:left;
}
QPushButton:hover
{
color: rgb(255, 255, 255);
}
</string>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel
{
background-color: transparent;
font: 14px &quot;Microsoft YaHei&quot;;
color:#B4C4D9;
border:0px;
}</string>
       </property>
       <property name="text">
        <string>版本：1.0.0.1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../CArmWorkStation.qrc"/>
 </resources>
 <connections/>
</ui>
